nuxt.config.ts

```diff
@@ -156,7 +156,8 @@ const config: NuxtConfig = {
   },
   'nuxt-purgecss': {
     mode: 'postcss',
-    enabled: ({ isDev, isClient }) => !isDev && isClient,
+    enabled: ({ isDev, isClient }: { isDev: boolean; isClient: boolean }) =>
```

Comment:
nuxt-purgecss v1.0.0から、`enabled`の型は`({ isDev: boolean; isClient: boolean }) => boolean`ではなく`boolean`ではありませんか？

https://github.com/Developmint/nuxt-purgecss#enabled
https://github.com/Developmint/nuxt-purgecss/blob/master/CHANGELOG.md#100-2020-04-26

実装を見ると、`enabled === false` ではなく`!enabled`となっているので、`enabled`の値が`false`を返す関数の場合、関数はtruthyなので常に`enabled`が有効な状態になりそうです。

https://github.com/Developmint/nuxt-purgecss/commit/4d37843f6738d4e1f7a889c8f12f115f13ac40a6

Reply:
おお、ご指摘ありがとうございます 🙏 
とすると、いままでずっと `enabled: true` の状態だったんでしょうか。
CHANGELOGをみると、そもそも `enabled` の指定は不要そうに見えますが、どうなんでしょう...？

Reply:
@kaizumaki [デフォルト設定](https://github.com/Developmint/nuxt-purgecss/blob/master/lib/utils.js)でいい場合は不要ですが、このプロジェクトの場合はデフォルト設定とは異なるので必要かと思います。

Reply:
```suggestion
    enabled: environment !== 'development' && process.client,
```
@munierujp なるほどです。
（ちょっと行がずれますが）このように置き換えられるでしょうか。
`environment !== 'development'` は `environment === 'production'` でいいような気もしますが、どうでしょうね？
`isClient` はnuxt v2.5.0でdeprecatedになっていたようです（ [参考](https://github.com/nuxt/nuxt.js/commit/29c3c4250c4daf225de85c2fdbfe519ebe706595) ）

Reply:
@kaizumaki 元の条件式に相当するコードはそうなりますね。
条件を `environment !== 'development'` とするか `environment === 'production'` とするかは、environmentが他の値（stagingなど）を取りうるのか、そうであればその値のときに有効にしたいかどうか次第かと思います。
たとえばstagingという値になりうる場合、stagingで有効化したい（developmentでのみ無効化したい）場合は前者に、無効化したい（productionでのみ有効化たい）場合は後者にすべきでしょう。

Reply:
@munierujp フォローありがとうございます 🙏 ここは `environment !== 'development'` でいきますね。